spyder_kernels/console/kernel.py

```diff
@@ -644,3 +632,55 @@ def _load_wurlitzer(self):
                 get_ipython().run_line_magic('reload_ext', 'wurlitzer')
             except Exception:
                 pass
+
+    def cmd_input(self, prompt=''):
```

Comment:
Please move this method, `pdb_input_reply` and `_interrupt_eventloop` to be below `set_pdb_execute_events` (just to put together all methods related to Pdb in the section of the class).